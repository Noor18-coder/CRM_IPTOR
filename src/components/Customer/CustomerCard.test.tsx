import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import BusinessPartnerCard from './CustomerCard';
import StoreMock from '../../mocks/Store.mock';
import BusinessPartnerListItemMock from '../../mocks/BusinessPartnerListItem.mock';

const store = StoreMock.createInitialStore();
const businesspartner = BusinessPartnerListItemMock.getBusinessPartnerListItemConstVal()[0];

describe('[Customer] BusinessPartnerCard', () => {
  it('should renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <BusinessPartnerCard businesspartner={businesspartner} />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
