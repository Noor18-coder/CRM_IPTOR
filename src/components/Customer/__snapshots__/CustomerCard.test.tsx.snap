// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Customer] BusinessPartnerCard should renders correctly 1`] = `
Array [
  <div
    className="d-flex justify-content-between title-row"
  >
    <div>
      <span
        className="customer-card-text"
      >
        3rd generation
      </span>
      <p
        className="area-text"
      >
        a
      </p>
    </div>
  </div>,
  <div
    className="d-flex justify-content-between owner-row"
  >
    <div
      className="lft-col p-0"
    >
      Owner 
      <span
        className="customer-card-text"
      >
        --
      </span>
    </div>
    <div
      className="rgt-col"
    >
      Industry 
      <span
        className="customer-card-text"
      >
        qui
      </span>
    </div>
  </div>,
  <div
    className="d-flex justify-content-between location-row"
  >
    <div
      className="lft-col p-0"
    >
      <span
        className="display-inline"
      >
         Active Opp - 
      </span>
       
      <span
        className="customer-card-text display-inline"
      >
        0
      </span>
    </div>
  </div>,
]
`;
